.github/workflows/user_test.yml

```diff
@@ -0,0 +1,32 @@
+# Install standard as end users would do in GitHub Actions.
+
+name: User Test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+permissions:
+  contents: read
+
+jobs:
+  user-test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [18.x, 20.x, "lts/*", latest]
+        command:
+          - "npm install standard"
+          - "yarn add standard"
+
+    steps:
+      - uses: actions/setup-node@v4
+        with:
+          node-version: latest
+      - run: |
+          ${{ matrix.command }}
+          standard --version || true  # Fails with: 'standard: command not found'
```

Comment:
If you install it locally, then you need to invoke it as `npx standard` or `./node_modules/.bin/standard`

If you want it to be available directly then you need to install it globally instead: `npm install -g standard`

Reply:
```suggestion
          node_modules/.bin/standard --version
```